doc/developers.rst

```diff
@@ -18,13 +18,23 @@ with more coverage.
 
 Building Documentation
 ----------------------
+The following is needed in order to build the documentation.
+
+*    ``pip install sphinx``
+*    ``pip install sphinx-gallery``
+*    ``pip install Image``
```

Comment:
@jasmainak sphinx for building docs, sphinx-gallery for compiling the .py examples, Image provides the graph thumbnails